integration-tests/test_rhc.py

```diff
@@ -11,24 +16,109 @@ def test_version(rhc, subtests):
         rhc_version = version.parse(version_string.split()[2])
         assert isinstance(rhc_version, version.Version) is True
 
+"""
+# rhc connect
+... it will connect a system to Red Hat services
+... it will prompt for username and password
+... it will inform a user about connection progress
 
-def test_rhc_register(external_candlepin, test_config, rhc, subtests):
+
+# rhc disconnect
+... it will disconnect from all Red Hat services
+... it will inform a user how it goes
+"""
+def test_rhc_connect_disconnect(external_candlepin, test_config, rhc, subtests):
     assert not rhc.is_registered
+    candlepin_config = partial(test_config.get,"candlepin")
     with subtests.test(msg="RHC connect"):
-        proc= rhc.connect(
-            username=test_config.get("candlepin", "username"),
-            password=test_config.get("candlepin", "password"),
+        proc = rhc.connect(
+            username=candlepin_config("username"),
+            password=candlepin_config("password"),
         )
         logging.info(f'result is {proc.stdout}')
         assert "Successfully connected to Red Hat!" in proc.stdout.decode()
         assert rhc.is_registered
 
     with subtests.test(msg="RHC disconnect"):
-        rhc.disconnect()
+        proc = rhc.disconnect()
+        proc_stdout = proc.stdout.decode()
         logging.info(f'result of disconnect task: {proc.stdout}')
         assert not rhc.is_registered
+        assert "Disconnected from Red Hat Subsription Management",\
+            "The application should inform a system is disconnected from Red Hat Subscription Management"
```

Comment:
like JC mentioned proc_stdout should be used in assertion, else it wont work as expected.

Reply:
fixed